easybuild/easyconfigs/o/OpenMPI/OpenMPI-4.1.1-GCC-10.3.0.eb

```diff
@@ -39,6 +44,10 @@ dependencies = [
     ('PMIx', '3.2.3'),
 ]
 
+preconfigopts = './autogen.pl --force && '
+
+configopts = '--with-cuda=internal '
```

Comment:
On second thought..

> Note that @Micket suggests to let `--with-cuda=internal` be a default, but that'd need an easyblock change

Given that it relies on a patch, to me is seems it actually seems like fine to me to have it in the easyconfig rather than trying to put some parts in the easyblock.
Just add a comment explaining this new option, and how you could opt-out of it (by using --without-cuda (and optionally remove the then unnecessary patch(es)))

Reply:
@bartoldeman Could we write a small comment along the lines of
```suggestion
# Patch and custom configure option can be removed if CUDA support isn't wanted.
configopts = '--with-cuda=internal '
```
to indicate that it is optional for those without CUDA hardware. I suppose all we need is to just remove the configopts and you'll have no CUDA support (or overhead), so, very simple change and i don't think we need any easyblock specialities